src/dapps/slice.ts

```diff
@@ -83,6 +87,14 @@ export const slice = createSlice({
       state.dappsListLoading = false
       state.dappsListError = action.payload.error
     },
+    favoriteDapp: (state, action: PayloadAction<FavoriteDappAction>) => {
+      state.favoriteDappIds = [...new Set([action.payload.dappId, ...state.favoriteDappIds])]
```

Comment:
Intuitively I would have added the new favorite at the end. If not already done, maybe it's worth checking with Reed?

Reply:
thanks for flagging this! i put the new dapps at the top because i was thinking about the recently used dapps / carousel on the home screen but the reverse also makes sense. i checked with reed and he said to follow whatever rule we have on the dapps screen now, so adding new dapps to the bottom of a section makes sense. i've updated the order